modules/apps/konsole.nix

```diff
@@ -132,6 +142,9 @@ in
             (lib.mapAttrs (keyName: keyAttrs: { value = keyAttrs; }) groupAttrs))
           cfg.extraConfig)
       )
+      {
+        "UiSettings"."ColorScheme".value = lib.mkIf (cfg.ui.colorScheme != null) cfg.ui.colorScheme;
```

Comment:
I need immutability here as well for this to not simply be overwritten when starting konsole for some reason.

Reply:
Sure, I will make that immutable.

Some more notes:

* I tried to run `home-manager switch` from inside konsole (checking my working theory, that konsole writes the current theme into the konsolerc upon closing the window), but that's not it.
* I do not have the same bug, so it is introduced somewhere where our systems differ. (I do not use overrideConfig, do you?)
* I would like to write down somewhere, where it's useful for future bug hunting. (Maybe an issue? Or a "known-weird-behaviour.md"?)

Reply:
Maybe it has something to do with using `overrideConfig`. I'm also using konsole version "24.05.0". The behavior is illustrated if I edit `konsolerc` manually by adding (initially having the default colorscheme):
```nix
[UiSettings]
ColorScheme=Krita darker
```
and then launch konsole, this will have changed to
```nix
[UiSettings]
ColorScheme=
```
(while some new lines will have been added). Setting the key to immutable will not have this effect.

Regarding writing it down, I reckon adding a comment in the code right above the `immutability = true;` part suffices, maybe something along the lines of "Immutability seems to be needed when using overrideConfig (see https://github.com/pjones/plasma-manager/pull/186 for details)" (if `overrideConfig` is indeed the issue)

Reply:
I changed something, if this doesn't fix it idk either.